script.js

```diff
@@ -0,0 +1,245 @@
+const form = document.forms[0];
```

Comment:
I recommend you to not use this approach. If you add another form you should keep in mind that index should be changed. Find it by class, id.

Reply:
`const form = document.querySelector('.form');`
Fixed